libbeat/scripts/Makefile

```diff
@@ -116,17 +116,17 @@ prepare-tests:
 # Race is not enabled for unit tests because tests run much slower.
 .PHONY: unit-tests
 unit-tests: prepare-tests
-	$(GOPATH)/bin/gotestcover -coverprofile=${COVERAGE_DIR}/unit.cov -short ${GOPACKAGES}
+	$(GOPATH)/bin/gotestcover -tags=unit $(RACE) -coverprofile=${COVERAGE_DIR}/unit.cov  ${GOPACKAGES}
```

Comment:
@urso I will remove the race flag again so we can get this PR in without fixing this. But I opened a separate issue to keep track on it: https://github.com/elastic/beats/issues/1074